thread/stack-allocator.cpp

```diff
@@ -28,7 +28,10 @@ limitations under the License.
 
 namespace photon {
 
-const static size_t PAGE_SIZE = getpagesize();
+inline uint32_t get_pagesize() {
```

Comment:
How about put this inline function `get_pagesize()` into thread.h file,
so that same code could not appear twice in both `stack-allocator.cpp` and `thread.cpp`?

Reply:
Talked to @lihuiba , he suggested if page_size being used somewhere else again, we encapsulate it.

Reply:
If it is used in a third place in the future, let's put it in a common header file, but not in thread.h, because it's conceptually unrelated to thread.